server/flaskr/database/database.py

```
@@ -0,0 +1,53 @@
+
+    @classmethod
+    def run_sql_file(cls, filename: str):
+        with cls._get_db() as conn:
+            cursor = conn.cursor()
+            path = os.path.join("flaskr", "database", "sql", f"{filename}.sql")
+
+            with open(path, "r", encoding="utf-8") as f:
+                cursor.execute(f.read())
+
+    @classmethod
+    def _get_db(cls):
```

Ruff (0.15.20)

[warning] Missing return type annotation for classmethod `_get_db`

(ANN206)